custom-dependencies/composer.json

```diff
@@ -7,7 +7,14 @@
             "url": "https://packages.drupal.org/8"
         }
     ],
-    "require": {},
+    "require": {
+        "mglaman/drupal-check": "^1.4",
+        "mglaman/phpstan-drupal": "^1.1",
+        "palantirnet/drupal-rector": "^0.15.1",
+        "phpstan/extension-installer": "^1.3",
+        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan-deprecation-rules": "^1.1"
+    },
```

Comment:
This needs to be in a different PR as it's a different matter from the purpose of this ticket.

Also, the concept of that change would cause conflicts and needs more refinement before being included. If we're going to remove this before a release, it will be something we need to remember to do... or automate.  I'd rather us keep it as it was - where those dependencies get installed, but not triggered for users unless the set it up - OR we could add a `dev-dependencies/composer.json` for these sorts of concerns... But all this belongs on another ticket and another PR.

Reply:
Oops. I'll remove. Sorry.